README.md

```diff
@@ -254,55 +268,45 @@ export default function RootLayout({ children }) {
 
 ### 2. Create StoryblokProvider and Import your Storyblok Components
 
-Create the `components/StoryblokProvider.jsx` file. Re-initalize the connection with Storyblok (this time, on the client) using `storyblokInit`, and import your Storyblok components:
+Create the `components/StoryblokProvider.jsx` file. Re-initalize the connection with Storyblok (this time, on the client) using the `getStoryblokApi` function imported from the `lib/storyblok` file. This will enable the client-side compoenents to interact with the Storyblok API, including the Visual Editor.
 
 ```js
-/** 1. Tag it as a client component */
-'use client'
-import { apiPlugin, storyblokInit } from '@storyblok/react/rsc'
+// components/StoryblokProvider.jsx
+'use client';
 
-/** 2. Import your components */
-import Page from '../components/Page'
-import Teaser from '../components/Teaser'
-
-/** 3. Initialize it as usual */
-storyblokInit({
-  accessToken: 'YOUR_ACCESS_TOKEN',
-  use: [apiPlugin],
-  components: {
-    teaser: Teaser,
-    page: Page,
-  },
-})
+import { getStoryblokApi } from '@/lib/storyblok';
 
 export default function StoryblokProvider({ children }) {
-  return children
+  getStoryblokApi();
+  return children;
 }
 ```
 
-> Note: it's necessary to re-initialize here as well, as to enable the live editing experience inside the Visual Editor you need to initialize the lib universally (client + server).
-
 ### 3. Fetch Content and Render Components
 
-The `getStoryblokApi` function, which is an instance of [storyblok-js-client](https://github.com/storyblok/storyblok-js-client) can be used to fetch the data from the Storyblok API. This should be imported from `@storyblok/react/rsc`.
-You can render the content of your route with the `StoryblokStory` component, which will automatically handle the Visual Editor live events when editing the story. In `app/page.jsx`, use them as follows:
+The `getStoryblokApi` function can now be used inside your Story components to fetch the data from Storyblok. In `app/page.jsx`, use it as follows:
 
 ```js
-import { getStoryblokApi, StoryblokStory } from '@storyblok/react/rsc'
+import { StoryblokClient, ISbStoriesParams } from '@storyblok/react';
+import { getStoryblokApi } from '@/lib/storyblok'; // Remember to import from the local file
```

Comment:
Can we move it down one line? Mainly to follow the usual js-standard of:
// First the libs imports
// Then the local files imports

Reply:
Sure!